web/repositories.go

```diff
@@ -50,3 +50,11 @@ func (s *QuickFeedService) isEmptyRepo(ctx context.Context, sc scm.SCM, request
 	}
 	return isEmpty(ctx, sc, repos)
 }
+
+func repoTypes(enrollment *qf.Enrollment) []qf.Repository_Type {
+	repositories := []qf.Repository_Type{qf.Repository_INFO, qf.Repository_ASSIGNMENTS, qf.Repository_USER, qf.Repository_GROUP}
+	if enrollment.IsTeacher() {
+		repositories = append(repositories, qf.Repository_TESTS)
+	}
```

Comment:
Add the following:
```go
if enrollment.GetGroupID() > 0 {
	repositories = append(repositories, qf.Repository_GROUP)
}
```
Given this code, we don't need the length comment explaining that `getRepo()` returns any (the first) group repository for the course.